.github/codecov.yml

```diff
@@ -16,4 +16,14 @@
 # under the License.
 
 ignore:
-  - "**/cucumber"
\ No newline at end of file
+  - "**/cucumber"
+coverage:
+  status:
+    project:
+      default:
+        target: auto
+        threshold: 5%
+    patch:
+      default:
+        target: 0%
+        
```

Comment:
Nit add newline.

Reply:
That was odd. It did have a newline but I guess the yml format neeeds 2 new lines in order for it to count?

Git warned me about it until I added two, so now there are two.